cfgov/legacy/templates/hud/housing_counselor.html

```diff
@@ -13,198 +13,255 @@
 
 {% block content %}
 <section>
-  <ul class="bread meta">
-    <li><a href="/">Home</a></li>
+    <ul class="bread meta">
+        <li><a href="/">Home</a></li>
         <li>Find a housing counselor</li>
-  </ul>
-  <div class="share mini">
-    <a href="http://www.facebook.com/sharer.php?u=http://www.consumerfinance.gov/find-a-housing-counselor/&amp;t=Find%20a%20housing%20counselor" class="share-facebook noStyles"><img src="{% static 'nemo/_/img/icon_small_facebook.png' %}" alt="Share on Facebook"></a>
-    <a href="http://twitter.com/intent/tweet/?url=http://www.consumerfinance.gov/find-a-housing-counselor/&amp;via=CFPB" class="share-twitter noStyles"><img src="{% static 'nemo/_/img/icon_small_twitter.png' %}" alt="Share on Twitter"></a>
-    <a href="http://api.addthis.com/oexchange/0.8/forward/email/offer?url=http://www.consumerfinance.gov/find-a-housing-counselor/&amp;title=Find a housing counselor&amp;pubid=ra-4da354ee346886d2" class="share-email noStyles"><img src="{% static 'nemo/_/img/icon_small_email.png' %}" alt="Share via email"></a>
-  </div>
-  <article class="page">
-
+    </ul>
+    <div class="share mini">
+        <a href="http://www.facebook.com/sharer.php?u=http://www.consumerfinance.gov/find-a-housing-counselor/&amp;t=Find%20a%20housing%20counselor" class="share-facebook noStyles">
```

Comment:
Since you're touching these templates, is it worth converting these links to https?

Reply:
Done! I converted them all on this page to https